src/vss_tools/vspec/vssexporters/vss2samm/vss2samm.py

```diff
@@ -10,35 +10,17 @@
 # Convert all vspec input files to a ESMF - Aspect Model Editor (SAMM) - ttl formatted file(s).
 #
 
-
-import importlib
 from pathlib import Path
 
 import rich_click as click
 import vss_tools.vspec.cli_options as clo
+import vss_tools.vspec.vssexporters.vss2samm.helpers.ttl_helper as ttl_helper
```

Comment:
Please check above comment.

These should be loaded after the Config is initialized, i.e. populated with user input for OUTPUT_NAMESPACE, VSPEC_VERSION and SPLIT_DEPTH fields.

Node, since SPLIT_DEPTH is defaulted to 1 in the Config.init(), maybe we can even default it to 1 at its initialization, instead of None.

Reply:
i will close the PR now since I have no time fixing the root cause which in my opinion is using values statically that are generated dynamically...